.vsts-dotnet.yml

```diff
@@ -26,6 +26,13 @@ stages:
   displayName: Build
 
   jobs:
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
When do you expect this job to run, exactly? I don't think this condition will ever fire in practice.

Reply:
Ah, this is a relic from dnceng builds. Thank you for catching it; will remove